src/components/entry/EntryNav.js

```diff
@@ -9,8 +9,8 @@ const EntryNav = () => {
   const { state } = useLocation()
 
   const onBackButtonClick = () => {
-    // Default to going back to the map. This occurs when the user opens /entry/{typeId} directly
-    history.push(state?.fromPage ?? '/map')
+    // Default to going back to the list. This occurs when the user opens /entry/{typeId} directly
+    history.push(state?.fromPage ?? '/list')
```

Comment:
Can you add a TODO here to create a utils file for routing? It seems like we're doing a lot of manual fromPage sets and I would prefer to eventually make a set of standardized helper functions to do this. Thanks.